app/forms/hyrax/forms/pcdm_collection_form_decorator.rb

```diff
@@ -1,7 +1,25 @@
 # frozen_string_literal: true
+# OVERRIDE Hyraxv5.0.0 to add the ability to upload a collection thumbnail
 
 Hyrax::Forms::PcdmCollectionForm.class_eval do
   include Hyrax::FormFields(:basic_metadata)
   include Hyrax::FormFields(:bulkrax_metadata)
   include CollectionAccessFiltering
+
+  ThumbnailInfoPrepopulator = lambda do |_options = nil|
```

Comment:
why is only thumbnail here. what about logo and banner?

Reply:
Logo and Banner live in Hyrax. We don't need to change or override it.